lwatools/plot/vis.py

```diff
@@ -32,7 +13,26 @@ def get_vis_indices(id_pairs):
 
     return indices
 
-        
+def vis_phase_scatter_3d(u, v, vis, l=None, m=None, model=point_source_visibility_model_uv):
```

Comment:
show vs save arg would be useful. Or maybe default show optional save. All of these things can be left for when they're used next though - you don't have to change anything. As these changes are desired we will add them.

Reply:
I'll leave this for now - it shouldn't be too hard to add when we need it